crates/jit/src/link.rs

```diff
@@ -132,6 +146,8 @@ cfg_if::cfg_if! {
         extern "C" {
             pub fn __rust_probestack();
         }
-        static PROBESTACK: unsafe extern "C" fn() = __rust_probestack;
+        static PROBESTACK: unsafe extern "C" fn() = empty_probestack;
```

Comment:
Is this a temporary workaround? If so, could you add a comment here explaining what the real fix is? If not, could you add a comment explaining why probing isn't needed? :-)

Reply:
Addressed -- it seems that stack probes are only supported on x86 / x86-64. I added a link to Rust's compiler-builtins impl of `__rust_probestack` where a comment describes the situation. Modified the code here to refer to `empty_probestack` only on non-x86, non-x86-64 architectures.

Reply:
FWIW with https://github.com/bytecodealliance/wasmtime/pull/1490 we won't need this anymore anyway.